conf/nomad/hive.hcl

```diff
@@ -59,16 +59,34 @@ job "${service_name}" {
         timeout = "90s"
       }
 
-      check {
+     /* check {
         name = "hive-minio-availability"
-        type     = "http"
-        path     = "/minio/health/ready"
-        port     = ${minio_local_bind_port}
+        type     = "script"
+        task     = "metastoreserver"
+        command  = "/usr/bin/curl"
+        args = [
+          "-s",
+          "-o",
+          "/dev/null",
+          "-w",
+          "HTTP code: %%{http_code}",
+          "127.0.0.1:${minio_local_bind_port}/minio/health/ready"
+        ]
         interval = "15s"
         timeout  = "5s"
-        address_mode = "driver"
       }
-
+      */
```

Comment:
Maybe I missed something while being a little in and out of the call, but why is this commented out?

Reply:
The test failed again on friday, so I guess Fredrik commented it out to isolate the error.

Reply:
If that's what it is, I'm cool with it too